requirements-base.txt

```diff
@@ -4,7 +4,7 @@ celery>=3.1.8,<3.1.19
 click>=5.0,<7.0
 # 'cryptography>=1.3,<1.4
 cssutils>=0.9.9,<0.10.0
-Django>=1.6.0,<1.7
+Django>=1.6.11,<1.9
```

Comment:
one concern is that onpremise users will upgrade to django 1.8 and we dont have a great way to lock them into 1.6 right now

Reply:
Can we keep this bit as it was? We can technically override in `setup.py` with an environment variable or something to say "Yes, give me 1.8".

My bigger concern with this personally is local dev. Every fresh install is going to get 1.8.x now when we're technically not ready for it.

Reply:
im fine with doing it in setup.py, but we could also just defer tagging a release until 1.8 is proven stable

Reply:
That won't help for local dev though. New employees, etc. They're all going to get 1.8.

Reply:
good point making the change